Grammar/python.gram

```diff
@@ -164,29 +164,33 @@ dotted_name[expr_ty]:
     | NAME
 
 if_stmt[stmt_ty]:
+    | invalid_if_stmt
```

Comment:
Just wondering, was this moved up for consistency with the other rules or is there another reason for it?

Reply:
Is moved so it can work. PEG alternatives are ordered and if this is not the first one the backtracking will make it fail before is even consider